setup.py

```diff
@@ -294,11 +294,63 @@ def adjust_installation_command():
                 else:
                     readme_lines[ln] = "  pip install Pootle\n"
 
+    def replace_urls():
+        from pootle.core.utils import version
+        branch_escape = version.get_git_branch().replace('/', '%2F')
+        for ln, line in enumerate(readme_lines):
+            for pattern, replace, rewrite_type in (
+                # Release Notes
+                ('releases/[0-9]\.[0-9]\.[0-9]\.html', 'releases/%s.html' %
+                 version.get_main_version(), 'all'),
```

Comment:
Nit: put each of the 3 values in the tuples here in a line of its own to increase readability.